Rear Rider/Rear Rider/StreamingAndRecording/WifiManager.swift

```diff
@@ -0,0 +1,31 @@
+// RearRider
+//
+// Calin Pescaru
+//
+// October 2022
+//
+// Bluetooth Manager
+//
+// Manages the Wi-Fi connection with the RPi.
+
+import Foundation
+
+/// Handles the Wi-Fi connection with the RPi. The commands are sent over BT
+class WifiManager: ObservableObject {
+    static var shared = WifiManager()
+    private var bleManager = BLEManager.shared
+    
+    @Published var wifiOn = false
```

Comment:
unless im missing it, it doesn't look like this gets toggled in this class anywhere? so it'll always stay false?

Reply:
It's toggled in the BLEManager.swift when it's reading the wifi characteristic. Maybe it's not the best programming approach. Should I encapsulate it in a function call?

Reply:
Ah, yeah I think it would make more sense to encapsulate it, even just in the functions you have now. No worries for now though; I figured I was overlooking it. Good for me to approve for now 👍🏻